base/timing.jl

```diff
@@ -154,18 +155,21 @@ julia> Base.format_bytes(1024)
 julia> Base.format_bytes(10000)
 "9.766 KiB"
 
+julia> Base.format_bytes(10000; sigdigits=1)
+"9.8 KiB"
```

Comment:
Am I missing something or these are two significant digits?  Compare with `round`:
```julia-repl
julia> round(10000 / 1024; sigdigits=1)
10.0

julia> round(10000 / 1024; sigdigits=2)
9.8
```

Reply:
Okay, the docs for `Ryu.writefixed` says:

> precision: minimum number of significant digits to be included in the decimal string; extra '0'
>       characters will be added for padding if necessary

So it is the number of sig digits of the decimals I guess.

Reply:
That's what `round` calls "digits":
```julia-repl
julia> round(10000 / 1024; digits=0)
10.0

julia> round(10000 / 1024; digits=1)
9.8

julia> round(10000 / 1024; digits=3)
9.766
```